forms/FormAction.php

```diff
@@ -100,7 +100,7 @@ public function Title() {
 
 		// Remove this method override in 4.0
 		$decoded = Convert::xml2raw($title);
-		if($decoded !== $title) {
+		if($decoded != $title) {
```

Comment:
Because we only want to deprecate if they are setting the title to something like `Left &amp; Main`, because `'Left & Main' !== 'Left &amp; Main'`. Not if they set it to false, and `false !== Convert::xml2raw(false)` or `null !== Convert::xml2raw(null)`.